conda/environment-dev.yaml

```diff
@@ -4,7 +4,7 @@ channels:
   - bioconda
   - defaults
 dependencies:
-  - r-base >= 3.5

```

Comment:
Is it necessary to remove that restriction in order to make it work?

Reply:
I initially removed the version and set strict conda-channel priority because the image took forever to solve conda env while building. Weirdly enough if I now try to build the package without these it works. So I put the r-base version back in c8f4fba